### DISCUSSION TITLE
Matter.js Controller "Successful Paired Devices"-Collection

### DISCUSSION THREAD

**Discussion by Apollon77** [matter-js/matter.js#316] [Matter compatible devices / controllers]
This thread should collect devices that were successfully paired with Matter.js as controller and can be operated.

**Comment by Apollon77** [comment]
I already paired the following:
* Eve Energy Smart Plugs (EU-Device, Thread - Commissioned via BLE into an own Thread-Network, but also a secondary Controller via Apple Hub)
* TP-Link/Tado  Mini Smart Wi-Fi Plugs P125M (US-Device, commissioned via BLE into Wi-Fi)
* Meross  Wi-Fi Plug Mini MS115MA (US-Device, commissioned via BLE into W-Fi)
* Meross Smart Wi-Fi Mini Plug MS315 (EU-Device, commissioned via BLE into Wi-Fi)

**Comment by JimBuzbee** [comment]
I have paired the following device which can be found under various branding on Aliexpress, Temu, etc.

- Smart Switch Module. Manufacturer: Uascent. Model: Smart-Plug-M028-US101. (US-Device, commissioned via BLE into Wi-Fi)

**Comment by Apollon77** [comment]
new additions:
* Wiz2 LED bulb (E27, commissioned via Wi-Fi after getting Matter code from Wiz App)
* Nanoleaf Essentials LED bulb (E27, commissioned via BLE into own Thread, also secondary controller via Wi-Fiafter paired to Apple Hub)

**Comment by Luligu** [comment]
I commissioned via apple hub:
- Eve motion
- Nanoleaf Smart Holiday String Lights.

When you say you have commissioned a device via Ble into Thread or WiFi what code you use exactly?

**Reply by Apollon77** [reply]
Ble commissioning is relevant if the devices are not commissioned before. Then you need BLE on your machine and you provide the credentials for - in cse of the two named devices - an own Thead network. Then BLE is used for this initial commissioning

**Reply by Luligu** [reply]
How can I commission directly with ble without using the Home app before? I have two new devices from Aqara/Xiaomi one is ble over wifi the other ble over thread. I want to try to commission them directly but I don’t know how.

**Reply by Apollon77** [reply]
for wifi: in fact see Controller example ... you need to add your wifi credentials to the commissioningOptions and import the matter-node-ble.js library which brings the relevant functionality - and having a BLE device on your host server. And bring the device near to that server because BLE has limited range.

for Thread: Basically the same but you need to provide Thread credentials .. but for this you need to get them - this means you need to have an own Thread network.  I used https://www.amazon.com/-/de/dp/B09DMK445K?psc=1&smid=A2GTSJRNFEVVSP&ref_=chk_typ_imgToDp ... then you flash OTBR on it and execute OTBR software in a VM or raspi ... with this you get an own thread network.

**Comment by Apollon77** [comment]
new Additions:
* Nuki SmartLock 4 Pro (Thread)
* Tuo Button (Thread)
* Onvis Plug (Wifi)
* Eve Door&Window (Thread)
* Siegenia Smart Window Handle (Thread)

**Comment by odegroote** [comment]
Ikea alpstuga temp/hum/co2/pm2.5 (Thread)
Ikea grillplats smartplug (Thread)
Ikea TimmerFlotte temp/hum (Thread, battery)

via Espressif Thread Border Router kit, using Homeassistant for commissioning
